public/docs/ts/latest/guide/architecture.jade

```diff
@@ -193,37 +186,32 @@ figure
 figure
   img(src="/resources/images/devguide/architecture/template.png" alt="Template" align="left" style="width:200px; margin-left:-40px;margin-right:10px" )
 :marked
-  We define a component's view with its companion **template**. A template is a form of HTML
+  You define a component's view with its companion **template**. A template is a form of HTML
   that tells Angular how to render the component.
 
-  A template looks like regular HTML much of the time ... and then it gets a bit strange. Here is a
+  A template looks like regular HTML much of the time but with some differences. Here is a
```

Comment:
Seems run-on and a bit wordy. Maybe:

A template looks like regular HTML, except for a few differences.